### PR TITLE
Update existing testdata (#1228)

### PR DESCRIPTION
**Pull request by TobiasNx**
To compare and discuss the new update

### CODE REVIEW

File: src/test/resources/alma/(DE-605)HT000161712.json [hunk 1]
```diff
@@ -46,88 +46,77 @@
...
     }
   }, {
-    "id" : "https://lobid.org/item/2398238870006445",
+    "id" : "https://lobid.org/item/23197423230006445",
     "type" : [ "ITM" ],
     "label" : "lobid Bestandsressource",
     "heldBy" : {
       "id" : "http://lobid.org/organisations/DE-290#!",
       "label" : "lobid Organisation"
     },
```
Comment (TobiasNx) [inline src/test/resources/alma/(DE-605)HT000161712.json:81]: missing callNumber

File: src/test/resources/alma/(DE-605)HT003176544.xml [hunk 2]
```diff
@@ -27,16 +28,6 @@
   <datafield tag="035" ind1=" " ind2=" ">
     <subfield code="a">(DE-605)HT003176544</subfield>
   </datafield>
```
Comment (TobiasNx) [inline src/test/resources/alma/(DE-605)HT003176544.xml:30]: OCLC ID in Datensatz weg

File: src/test/resources/alma/(DE-605)HT020202475.xml [hunk 3]
```diff
@@ -137,16 +141,6 @@
...
-  <datafield tag="982" ind1=" " ind2=" ">
-    <subfield code="a">Critical theory</subfield>
-    <subfield code="9">LOCAL</subfield>
-    <subfield code="M">49HBZ_UBD</subfield>
-  </datafield>
-  <datafield tag="982" ind1=" " ind2=" ">
-    <subfield code="a">Electronic books</subfield>
-    <subfield code="9">LOCAL</subfield>
-    <subfield code="M">49HBZ_UBD</subfield>
-  </datafield>
```
Comment (TobiasNx) [inline src/test/resources/alma/(DE-605)HT020202475.xml:149]: @hagbeck local keywords were deleted with the update?

**Comment by TobiasNx** [comment]
@dr0i we need the updated testdata to continue developing the ETL

**Comment by dr0i** [comment]
Added lost resources, merged into master, closed.

**Pull request merged**